receiver/k8sclusterreceiver/internal/service/service.go

```diff
@@ -24,6 +24,11 @@ func Transform(service *corev1.Service) *corev1.Service {
 	}
 }
 
+// Maximum number of services assigned to a pod that will be added as properties.
+// If a pod has more services than this limit, it's typically an ingress controller or a similar service.
+// In that case, we don't set any service properties to the pod since it's not useful.
+const podServiceLimit = 10
```

Comment:
Could it be that we have an ingress controller with fewer Services? Wouldn't that be an inconsistency where some ingress controller Pods come with service properties while others not?

Maybe we could just cap the amount of Services added instead of not adding them at all 🤔?